CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
+- `SpatialPingComponent` can now also report average ping measurements over a specified number of recent pings. You can specify the number of measurements recorded in `PingMeasurementsWindowSize` and get the measurement data by calling `GetAverageData`. There is also a delegate `OnRecordPing` that will be broadcast whenever a new ping measurement is recorded.
 - The Spatial Output Log window now displays deployment startup errors.
-- Added `bEnableClientQueriesOnServer` (defaulted false) which makes the same queries on the server as on clients if the unreal load balancer is enabled. Enable this to avoid clients seeing entities the server does not if the server's interest query has not been configured correctly.
-- Added log warning when AddPendingRPC fails due to ControllerChannelNotListening.
-- When running with Offloading enabled, Actors will have local authority (ROLE_Authority) on servers for longer periods of time to allow more native Unreal functionality to work without problems.
-- When running with Offloading enabled, and trying to spawn Actors on a server which will not be the Actor Group owner for them, an error is logged and the Actor is deleted.
-- The GDK now uses SpatialOS runtime version 14.5.1 by default.
-- Config setting `bPreventAutoConnectWithLocator` has been renamed to `bPreventClientCloudDeploymentAutoConnect`. It has been moved to GDK Setting. If using this feature please update enable the setting in GDK Settings.
-- USpatialMetrics::WorkerMetricsRecieved was made static.
-- Added the ability to connect to a local deployment when launching on a device by checking "Connect to a local deployment" and specifying the local IP of your computer in the Launch dropdown.
-- The Spatial GDK now default enables RPC Ring Buffers, and the legacy RPC mode will be removed in a subsequent release.
-- The `bPackRPCs` property has been removed, and the flag `--OverrideRPCPacking` has been removed.
-- Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
-- You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.
-- Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.
-- Added `OnSpatialPlayerSpawnFailed` delegate to `SpatialGameInstance`. This is helpful if you have established a successful connection but the server worker crashed.
+- We've added `bEnableClientQueriesOnServer` (default false) which makes the same queries on the server as it makes on clients, if the unreal load balancer is enabled. Enable this to avoid a situation in which clients receive updates about entities that the server doesn't receive updates about (if the server's interest query is configured incorrectly).
```

Comment:
```suggestion
- We've added `bEnableClientQueriesOnServer` (default false) which makes the same queries on the server as it makes on clients, if the GDK for Unreal's load balancer is enabled. Enable `bEnableClientQueriesOnServer` to avoid a situation in which clients receive updates about entities that the server doesn't receive updates about (which happens if the server's interest query is configured incorrectly).
```

Rephrased slightly - please check that the second sentence is accurate. (Previously it was: Enable this to avoid clients seeing entities the server does not if the server's interest query has not been configured correctly.)

Reply:
Looks good